app/posts/manage-teacher-training-applications/2022-07-13-improving-the-email-sent-to-candidates-about-new-interviews.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Improving the email sent to candidates about new interviews
+description: We now include the course subject and code. We also show details separately rather than in a single section.
+date: 2022-07-13
+---
+
+{% from "email/macro.njk" import appEmail %}
+
+The email sent to candidates when an interview is set up does not currently include the course name or code.
+
+We wanted to add these to the email and also reflect [changes we made to the email sent when an interview is updated](https://bat-design-history.netlify.app/manage-teacher-training-applications/emailing-candidates-about-upcoming-interviews-when-their-course-is-changed/).
```

Comment:
Use relative links, so that it works locally (and if we ever change domain in future).

```suggestion
We wanted to add these to the email and also reflect [changes we made to the email sent when an interview is updated](/manage-teacher-training-applications/emailing-candidates-about-upcoming-interviews-when-their-course-is-changed/).
```